### PR TITLE
#10: Info page content

### LINKED ISSUE #10: Create an info page
As we want everyone to be able to use the service we should have a descriptive info page on how to use the service. This would include a hands-on tutorial on how to export your data with pictures for each Finnish bank that we are supporting. 

Additionally we need to explain how the service works and that it is not saving the data anywhere else than in the users browser

However, the other pages can also include some instructions so that the info page does not get too cluttered. E.g. the export and import page can have information about specific steps needed for just that, importing and exporting

### PR DESCRIPTION
**Pull request by tu-tran**
Fixes #10

### CODE REVIEW

**Review by andersnylund — COMMENTED** [review]
I would still like to have screenshots from different banks and how to export your data from there. I'm not sure yet how that would be done the best way.

I also found a "bug" where the page is wider than the AppBar:
screenshot 2018-11-15 at 13 37 35
so if you scroll all the way to the right, there is a white space left to the AppBar

File: src/components/InfoSteps.js [hunk 1]
```diff
@@ -42,6 +43,9 @@ const styles = theme => ({
     display: 'inline-block'
   },
   instructions: {
+    width: '100%',
+    marginLeft: theme.spacing.unit * 3,
+    marginRight: theme.spacing.unit * 3,
     marginTop: theme.spacing.unit,
     marginBottom: theme.spacing.unit
   },
```
[the remark this answers is not in the document]
Comment (tu-tran) [reply src/components/InfoSteps.js]: Hmm, unfortunately Google gave me nothing, and I am not a customer of both those banks... Do you have a screenshot there ?

[the remark this answers is not in the document]
Comment (tu-tran) [reply src/components/InfoSteps.js]: > I would still want that we wouldn't use material-ui's way of styling with class based styling. Instead I would like to have it purely done with styled-components. But that is only my preference

Eew, in my experience, not following theme unit is a pain later on to support multiple themes (but not sure if its the case fore web dev though)... Anyway, for this kind of project, I think either way is fine so did not really think twice about this

[the remark this answers is not in the document]
Comment (andersnylund) [reply src/components/InfoSteps.js]: > Eew, in my experience, not following theme unit is a pain later on to support multiple themes (but not sure if its the case for web dev though)... Anyway, for this kind of project, I think either way is fine so did not really think twice about this

Could use the Context API for that 😏 But let's go with this for now. I just find the styled-components so much more readable

**Comment by andersnylund** [comment]
Didn't even think about it, but now I realized that having screenshots on the info page is not that great of an idea. If we need to change anything, we will have to update the screenshots each time

**Comment by tu-tran** [comment]
> Didn't even think about it, but now I realized that having screenshots on the info page is not that great of an idea. If we need to change anything, we will have to update the screenshots each time

yeah :D I was trying to do the "interactive guide" , where I could pop the arrow and show preview of real React component. Unfortunately, my dummy web skill get stuck after a few days :( So I went with the quick and dirty solution :(

**Review by andersnylund — APPROVED** [review]

**Review by andersnylund — CHANGES_REQUESTED** [review]
Changing my mind

**Review by andersnylund — APPROVED** [review]

**Pull request merged**